documentation/API_documentation/QoSProfile_Mapping_Table.md

```diff
@@ -1,25 +1,25 @@
 # QoS Profiles Mapping Table (REFERENCE DRAFT)
 
-Here are two possible examples of mapping tables from QoS Profiles to provided connectivity characteristics. The second example describes a case, where the network is specially prepared in well define geographical areas for supporting Quality on Demand needs.
+Here are two possible examples of mapping tables from QoS Profiles to provided connectivity characteristics. The intention of the examples is to illustrate a different usages of the defined QoS Profile labels. The first example focuses on a wide area network usage. The second example describes a case, where the network is specially prepared in well define geographical areas for supporting Quality on Demand needs.
 
 Example 1: Considering wide-area network usage.
 | QOS Profile labels | Network Service Description | Network Service Constraints |
 |----------| --------- | ----------- |
-| QOS\_E   | Latency stays stable under congestion (throughput upto 2Mbps). The application bitrate should not exceed 2Mbps. | The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
-| QOS\_L   | DL upto 100Mbps (unlimited?). The 5G System throughput is prioritized up to 100Mbps. At high load, the throughput is capped at 100Mbps and can be degraded to lower effective throughput (without a minimum level).  | The service is only provided, when the UE is located within the home network. |
-| QOS\_M   |DL upto 30Mbps. The 5G System throughput is prioritized up to 30Mbps. At high load, the throughput is capped at 30Mbps and can be degraded to lower effective throughput (without a minimum level). | The service is only provided, when the UE is located within the home network. |
-| QOS\_S   | DL upto 10Mbps. The 5G System throughput is dprioritized up to 10Mbps. At high load, the throughput is capped at 10Mbps and can be degraded to lower effective throughput (without a minimum level). | The service is only provided, when the UE is located within the home network. |
+| QOS\_E   | Latency stays stable under congestion (throughput upto 500kbps). The application bitrate should not exceed 500kbps. | The service is only provided, when the UE is located within the geographical area, according to the agreement with the CSP.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
+| QOS\_L   | The 5G System throughput is prioritized up to 20Mbps. At high load, the throughput is capped at 20Mbps and can be degraded to lower effective throughput (without a minimum level).  | The service is only provided, when the UE is located within the home network.<br>The access to the network service is restricted to a maximal number of simultaneous session.|
```

Comment:
"The access to the network service is restricted to a maximal number of simultaneous session." is currently added only to QOS_E and QOS_L ... but should be consequently also added to QOS_M and QOS_E

Reply:
ok, I added this. thx,